mmv1/templates/terraform/examples/transcoder_job_basic.tf.erb

```diff
@@ -0,0 +1,91 @@
+resource "google_transcoder_job" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['job_name'] %>"
+  template_id = google_transcoder_job_template.default.name
+  location    = "us-central1"
+
+  labels = {
+    "label" = "key"
+  }
+}
+
+resource "google_transcoder_job_template" "default" {
+  job_template_id = "<%= ctx[:vars]['job_template_id'] %>"
+  location = "us-central1"
+  config {
+    inputs {
```

Comment:
@bschaatsbergen  job_template is passing! the basic test fails due to a missing field in the config block.

From the logs:
```hcl
2024/08/06 21:15:49 [DEBUG] [transport] [server-transport 0xc001692480] loopyWriter exiting with error: transport closed by client 
    vcr_utils.go:152: Step 1/2 error: Error running apply: exit status 1
        
        Error: Error creating Job: googleapi: Error 400: config.input[0].uri missing
        
          with google_transcoder_job.default,
          on terraform_plugin_test.tf line 2, in resource "google_transcoder_job" "default":
           2: resource "google_transcoder_job" "default" {
```

documentation regarding the uri field: https://cloud.google.com/transcoder/docs/reference/rest/v1/JobConfig#Input

Reply:
Fixed in 95781a3 using a test fixture